### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by FK78**
Resolves #2587, #2588, #2589, #2590, #2591, #2592, #2593, #2594, #2595, #2596, #2597, #2598, #2599, #2600, #2601, #2602, #2603

**Overall change:** _Merging dependabot PRs._

**Code changes:**

- _`package.json`, `package-lock.json` and `CHANGELOG.md` bumped._
- _`CHANGELOG.md` major bump because dependencies have breaking change._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by thekp** [comment]
breaking change from chalk: https://github.com/chalk/chalk/releases
breaking change from stripe-ansi: https://github.com/chalk/strip-ansi/releases

I'm not sure if this should be a breaking version bump for Psammead, would appreciate some code owners opinion.

^ @jroebu14 @sareh @jblaut

**Review by jroebu14 — APPROVED** [review]

**Review by sareh — CHANGES_REQUESTED** [review]
There's a breaking change in `strip-ansi`: Looking at the release notes on their github, the recommended change is this:

```
Only use a CommonJS export for the TypeScript definition
You need to change import stripAnsi from 'strip-ansi'; to import stripAnsi = require('strip-ansi');
```

**Comment by FK78** [comment]
> There's a breaking change in `strip-ansi`: Looking at the release notes on their github, the recommended change is this:
> 
> ```
> Only use a CommonJS export for the TypeScript definition
> You need to change import stripAnsi from 'strip-ansi'; to import stripAnsi = require('strip-ansi');
> ```

We do not use `import stripAnsi` anywhere.

File: package-lock.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "requires": true,
```
Comment (j-pendlebury) [inline package-lock.json:5]: version needs bumping on line 3. Same with `package.json`

**Review by j-pendlebury — CHANGES_REQUESTED** [review]
CHANGELOG got updated but not the version in Psammead's package.json and package-lock

**Review by sareh — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Pull request merged**